### ISSUE TITLE
only the first record of type MyEnum is shown as "required".

### ISSUE THREAD

**Issue by ghuser** [asyncapi/avro-schema-parser#240]
**Description**

In the following asyncapi document:

```yml
asyncapi: 2.6.0
info:
  title: AsyncAPI
  version: 1.0.0
  description: AsyncAPI
channels:
  example:
    publish:
      message:
        $ref: '#/components/messages/example_message'
components:
  messages:
    example_message:
      name: example_event
      schemaFormat: application/vnd.apache.avro;version=1.9.0
      payload:
        type: record
        name: ParentRecord
        fields:
          - name: r1
            type:
              type: enum
              name: MyEnum
              symbols:
                - A
                - B
                - C
          - name: r2
            type: MyEnum
          - name: r3
            type: MyEnum

```

the preview is the one below:

![image](https://github.com/asyncapi/asyncapi-react/assets/672768/c4f447e3-7505-4a7a-977c-f2f6bdb5b39b)

As you can see in the image above only `r1` is shown as `required`.

**Expected result**

All of `r1`, `r2` and `r3` should be shown as `required`.

**Actual result**

`r2` and `r3` are not shown as `required`.

**Steps to reproduce**

Copy the above yml and paste it inside: [asyncapi.github.io/asyncapi-react/](https://asyncapi.github.io/asyncapi-react/) or  [studio.asyncapi.com](https://studio.asyncapi.com/) to see the preview.

**Troubleshooting**

**Comment by derberg** [comment]
@M3lkior @dalelane I think this one is very much related to Avro really and conversion to JSON Schema. Can you help?

**Comment by dalelane** [comment]
sorry for the delay in getting to this one!

I've had an attempt at a possible fix in https://github.com/asyncapi/avro-schema-parser/pull/246 

Is anyone able to take a look at that for me, please?

**Comment by dalelane** [comment]
This should be fixed in 3.0.15
